Assets/Mirror/Editor/Weaver/Processors/RpcProcessor.cs

```diff
@@ -81,6 +81,8 @@ public static MethodDefinition ProcessRpcCall(TypeDefinition td, MethodDefinitio
             rpcWorker.Append(rpcWorker.Create(OpCodes.Ldc_I4, NetworkBehaviourProcessor.GetChannelId(ca)));
             rpcWorker.Append(rpcWorker.Create(OpCodes.Callvirt, Weaver.sendRpcInternal));
 
+            NetworkBehaviourProcessor.WriteRecycleWriter(rpcWorker);
```

Comment:
@vis2k  a few lines up,  there was already a call to WriteCreateWriter, which now generates the call to `NeworkWriterPool.GetWriter()`